solrman/smstorage/zk_storage_test.go

```diff
@@ -120,39 +131,41 @@ func TestZkStorage_IsDisabled(t *testing.T) {
 	s, testutil := setup(t)
 	defer testutil.teardown()
 
-	if s.IsDisabled() {
+	if isDisabled, reason := s.IsDisabled(); isDisabled {
 		t.Error("expected to not be disabled")
 	}
 
-	testutil.create(s.disabledPath())
+	testutil.createWithData(s.disabledPath(), "testDisabled")
 
-	if !s.IsDisabled() {
+	if isDisabled, reason := s.IsDisabled(); !IsDisabled {
```

Comment:
we should start to prefer fstesting constructs `fstesting.Equals(t, expected, actual, "msg", args...)`

Reply:
Yeah. I'm  likely to redo a  chunk of  this with per-org splits/moves etc, so I'll look into refactoring the tests as a part of that :)

Reply:
might be blasphemous but should this even be open source? like it has a bunch of stuff specific to us, i dunno how many people are even using this

Reply:
I definitely feel like it's way too us-specific to be useful to many folks, and I don't think anyone else uses it really.